wgsl/index.bs

```diff
@@ -17795,40 +17795,184 @@ fn atomicLoad(atomic_ptr: ptr<AS, atomic<T>, read_write>) -> T
 Returns the atomically loaded the value pointed to by `atomic_ptr`.
 It does not [=atomic modification|modify=] the object.
 
-### Atomic Store ### {#atomic-store}
+### `atomicStore` ### {#atomic-store}
 
 ```wgsl
 fn atomicStore(atomic_ptr: ptr<AS, atomic<T>, read_write>, v: T)
 ```
 
 Atomically stores the value `v` in the atomic object pointed to by `atomic_ptr`.
 
-### Atomic Read-modify-write ### {#atomic-rmw}
+### Atomic Read-modify-write Arithmetic and Logical Functions ### {#atomic-rmw}
+
+Each function performs the following steps atomically:
+
+1. Load the original value pointed to by `atomic_ptr`.
+2. Obtains a new value by performing the operation (e.g. max) from the function
+    name with the value |v|.
+3. Store the new value using `atomic_ptr`.
+
+Each function returns the original value stored in the atomic object before the operation.
+
+#### `atomicAdd` #### {#atomic-add}
 
 ```wgsl
 fn atomicAdd(atomic_ptr: ptr<AS, atomic<T>, read_write>, v: T) -> T
+```
+
+Atomically performs an addition operation on the atomic object pointed to by `atomic_ptr`
+with the value `v`, and returns the original value stored in the atomic object before the operation.
+
+<div class='example wgsl global-scope' heading='Operation of `atomicAdd` as a function'>
```

Comment:
I notice in the HTML render that the backquotes aren't changing the text of the function name. Maybe change these to atomic addition.

For `atomicCompareExchangeWeak`, I'd probably drop the weak part (so atomic compare exchange).

Reply:
@alan-baker Makes sense. Updated, PTAL. Appreciated.